sdk/resourcemanager/Azure.ResourceManager/src/Generated/Resources/Models/ResourceIdentityType.Serialization.cs

```diff
@@ -9,8 +9,13 @@
 
 namespace Azure.ResourceManager.Resources.Models
 {
-    internal static partial class ResourceIdentityTypeExtensions
+    /// <summary>
+    /// Extension methods for ResourceIdentityType serialization
+    /// </summary>
+    public static partial class ResourceIdentityTypeExtensions
```

Comment:
internal to public because?

Reply:
It will be used in Resources SDK, previously Resources SDK will generate its own ResourceIdentityType.Serialization.cs, but now since they use the same namespace, we only keep the one in `Azure.ResourceManager` package and do not generate it in Resources SDK.
In this PR https://github.com/Azure/autorest.csharp/pull/1468, to avoid generating the same model class under the same `Azure.ResourceManager.Resources.Models` namespace, I added https://github.com/Azure/autorest.csharp/blob/cd22cf5a06132ea593eaded77e11cbd67638fd90/src/AutoRest.CSharp/Mgmt/AutoRest/MgmtTarget.cs#L32-L36 to skip generating the same model if it already exists in `Azure.ResourceManager`.
```
if (context.SourceInputModel?.FindForType(model.Declaration.Namespace, model.Declaration.Name) != null)
{
    continue;
}
```